hack/generate.sh

```diff
@@ -6,11 +6,6 @@ RED='\033[0;31m'
 GREEN='\033[0;32m'
 RESET='\033[0m'
 
-if ! [ -x "$(command -v goimports)" ]; then
```

Comment:
ooi, what did this do and why is it obsolete?

Reply:
We install it elsewhere, so this ended up being redundant.